icon_validator/rules/workflow_validators/workflow_screenshot_validator.py

```diff
@@ -99,38 +124,59 @@ def validate_screenshot_files_exist(self, spec):
             for file_name in os.listdir(f"{directory}/screenshots"):
                 self._files_list.append(file_name)
         except FileNotFoundError:
-            raise ValidationException(f"The screenshots directory could not be found at: {directory}\n"
-                                      "Please ensure that the screenshots directory exists.")
+            raise ValidationException(
+                f"The screenshots directory could not be found at: {directory}\n"
+                "Please ensure that the screenshots directory exists."
```

Comment:
f strings don't mix with regular strings. 

```suggestion
                f"Please ensure that the screenshots directory exists."
```

Reply:
<img width="319" alt="Screen Shot 2020-04-21 at 2 26 06 PM" src="https://user-images.githubusercontent.com/32079048/79905583-15341500-83dc-11ea-8397-426bc01332dd.png">
It works :)